recipes/cppcmd/all/test_package/main.cpp

```diff
@@ -0,0 +1,24 @@
+#include <cassert>
+#include <sstream>
+
+#include <cppcmd/CommandInterpreter.hpp>
+
+using namespace cppcmd;
+using namespace std;
+
+int main() {
+    unsigned int fooCalls = 0;
+    unsigned int barCalls = 0;
+
+    stringstream input;
+    auto interpreter = CommandInterpreter(input, std::cout);
+    interpreter.registerCommand("foo", [&](const auto&, auto&) { fooCalls++; });
+    interpreter.registerCommand("bar", [&](const auto&, auto&) { barCalls++; });
+
+    input << "foo\nbar\nundefined\nfoo\n";
+    interpreter.run();
+
+    assert(fooCalls == 2);
+    assert(barCalls == 1);
```

Comment:
for future reference, the test_package is to test the package not the library so these can be left out next time

Regardless 🏆 great work!